enterprise/reporting/flusher/aggregator/tracked_users_inapp.go

```diff
@@ -0,0 +1,119 @@
+package aggregator
+
+import (
+	"context"
+	"database/sql"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/segmentio/go-hll"
+
+	"github.com/rudderlabs/rudder-go-kit/config"
+	"github.com/rudderlabs/rudder-go-kit/stats"
+)
+
+type TrackedUsersInAppAggregator struct {
+	db    *sql.DB
+	stats stats.Stats
+
+	reportsCounter    stats.Measurement
+	aggReportsCounter stats.Measurement
+}
+
+func NewTrackedUsersInAppAggregator(db *sql.DB, s stats.Stats, conf *config.Config, table, module string) *TrackedUsersInAppAggregator {
+	t := TrackedUsersInAppAggregator{db: db, stats: s}
+	tags := stats.Tags{
+		"instance": conf.GetString("INSTANCE_ID", "1"),
+		"table":    table,
+		"module":   module,
+	}
+	t.reportsCounter = t.stats.NewTaggedStat("reporting_flusher_get_reports_count", stats.HistogramType, tags)
+	t.aggReportsCounter = t.stats.NewTaggedStat("reporting_flusher_get_aggregated_reports_count", stats.HistogramType, tags)
+	return &t
+}
+
+func (a *TrackedUsersInAppAggregator) Aggregate(ctx context.Context, start, end time.Time) (jsonReports []json.RawMessage, err error) {
+	selectColumns := "reported_at, workspace_id, source_id, instance_id, userid_hll, anonymousid_hll, identified_anonymousid_hll"
+	query := fmt.Sprintf(`SELECT %s FROM tracked_users_reports WHERE reported_at >= $1 AND reported_at < $2 ORDER BY reported_at`, selectColumns)
```

Comment:
Either pass tableName and making dynamic or define it as const and use it for both queries and stats.
```suggestion

const tableName = `tracked_users_reports`

func NewTrackedUsersInAppAggregator(db *sql.DB, s stats.Stats, conf *config.Config, module string) *TrackedUsersInAppAggregator {
	t := TrackedUsersInAppAggregator{db: db, stats: s}
	tags := stats.Tags{
		"instance": conf.GetString("INSTANCE_ID", "1"),
		"table":    tableName,
		"module":   module,
	}
	t.reportsCounter = t.stats.NewTaggedStat("reporting_flusher_get_reports_count", stats.HistogramType, tags)
	t.aggReportsCounter = t.stats.NewTaggedStat("reporting_flusher_get_aggregated_reports_count", stats.HistogramType, tags)
	return &t
}

func (a *TrackedUsersInAppAggregator) Aggregate(ctx context.Context, start, end time.Time) (jsonReports []json.RawMessage, err error) {
	query := `SELECT reported_at, workspace_id, source_id, instance_id, userid_hll, anonymousid_hll, identified_anonymousid_hll FROM ` + tableName + `  WHERE reported_at >= $1 AND reported_at < $2 ORDER BY reported_at` 
```